pallets/file-system/src/lib.rs

```diff
@@ -234,8 +240,20 @@ pub mod pallet {
         /// Revoke storage request
         #[pallet::call_index(2)]
         #[pallet::weight(Weight::from_parts(10_000, 0) + T::DbWeight::get().writes(1))]
-        pub fn revoke_storage_request(_origin: OriginFor<T>) -> DispatchResult {
-            todo!()
+        pub fn revoke_storage_request(
```

Comment:
This should actually iniciate a delete request, in case some BSPs or the MSP already confirmed the storage request and added the file to their tries. We have to make sure they deleted it. Please add it as a TODO at least

Reply:
yes will do, kindof added this quickly in the pr just to have something there.